packages/aws-lambda-otel-extension/test/utils/get-extension-server-mock.js

```diff
@@ -109,8 +109,8 @@ module.exports = (emitter) => {
                 state: 'Subscribed',
                 types: data.types,
               },
-            }
-          ).then((res) => res.text());
+            },
+          ]).then((res) => res.text());
```

Comment:
Did you mean to change the second param to an array?

Reply:
Yes. It's then passed as logs payload, and and AWS side in all cases it's an array.

When I was refactoring I observed that due this not being an array, we had one failure that was muted by try/catch clause which wasn't picking this up.

Reply:
Ahh okay nice catch 👍